html-forms/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Forms</title>
+  </head>
+  <body>
+
+    <h1>Coding Form!</h1>
+
```

Comment:
You are missing the `<form>` element. All of the `<input>` & `<textarea>` should be wrapped inside of your `<form>` element